priv/repo/migrations/20240509181234_change_days_of_week_column_type.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Screenplay.Repo.Migrations.ChangeDaysOfWeekColumnType do
+  use Ecto.Migration
+
+  def change do
+    execute(&execute_up/0, &execute_down/0)
+  end
+
+  defp execute_up do
+    repo().query!(
+      "ALTER TABLE pa_message ALTER COLUMN days_of_week TYPE integer[] USING (days_of_week::integer[])"
+    )
+  end
+
+  defp execute_down, do: nil
```

Comment:
**nibble (non-blocking):** any reason to not pass this SQL to `execute` directly?

```suggestion
  def change do
    execute "ALTER TABLE pa_message ALTER COLUMN days_of_week TYPE integer[] USING (days_of_week::integer[])", ""
  end
```

Reply:
I think this was a case of me tunneling on the first example I found for this function 😅 Got this cleaned up.